lib/routes.js

```diff
@@ -143,10 +130,25 @@ function validationHandler (validationError) {
 }
 
 module.exports = async function (app, opts) {
+  const errorFormatter = typeof opts.errorFormatter === 'function' ? opts.errorFormatter : defaultErrorFormatter
+
   if (typeof opts.errorHandler === 'function') {
     app.setErrorHandler(opts.errorHandler)
   } else if (opts.errorHandler === true || opts.errorHandler === undefined) {
-    app.setErrorHandler(defaultErrorHandler)
+    app.setErrorHandler((error, _, reply) => {
+      /* istanbul ignore if */
```

Comment:
I'm not sure about the inclusion of this here.

I see why it was added to the error handler (and this is where I've added it back here), but the `errorFormatter` can now override the status code of an error.

Reply:
Let's remove it.